packages/sdk/android/src/main/java/com/primerioreactnative/PrimerRNHeadlessUniversalCheckout.kt

```diff
@@ -46,6 +46,7 @@ class PrimerRNHeadlessUniversalCheckout(
         if (settingsStr.isNullOrBlank()) PrimerSettingsRN() else json.decodeFromString(
           settingsStr
         )
+      println(settings)
```

Comment:
Is this debugging leftover, do do we keep it?

Reply:
yeah, just testing due to issues with local build